source/Nevermore/IWriteQueryExecutor.cs

```diff
@@ -92,6 +92,38 @@ public interface IWriteQueryExecutor : IReadQueryExecutor
         /// <param name="cancellationToken">Token to use to cancel the command.</param>
         Task UpdateAsync<TDocument>(TDocument document, UpdateOptions options, CancellationToken cancellationToken = default) where TDocument : class;
 
+        /// <summary>
```

Comment:
I think this more explicit interface is nice. This is breaking change to the public API so should be versioned appropriately.